docs/Module Developer's Guide.md

```diff
@@ -970,4 +978,27 @@ enum CPU_SUBTYPE_ARM {
 With the enums above you can refer to `macho.CPU_TYPE_X86` and instead of 
 `macho.CPU_TYPE.CPU_TYPE_X86` and `macho.CPU_SUBTYPE_INTEL.CPU_SUBTYPE_I386`. 
 
+## Tests
+You'll notice that each module in `/yara-x/src/modules/` has a `tests/` directory with a nested `testdata/` directory. The testing framework is expecting a particular format and input structure to use them:
+1. File conveted to [Intel Hex](https://developer.arm.com/documentation/ka003292/latest/) format format
+2. Intel Hex format output zipped
+3. End file named <sha_256>.in.zip
+   
+### Structuring Testdata Input
+To convert the binary to Intel Hex format, we can use the scripts provided at [python-intelhex/intelhex](https://github.com/python-intelhex/intelhex/).
```

Comment:
A more straightforward alternative for generating the Intel Hex file (at least on Linux and Mac) is:

`objcopy -I binary -O ihex myfile myfile.in`

`objcopy` is usually present on Linux and Mac OS systems. In Mac OS it is named `llvm-objcopy`, though.

Reply:
Implemented the specific platforms 😄